docs/programming_guide/workflows_and_controllers.rst

```diff
@@ -0,0 +1,21 @@
+#########################
+Workflows and Controllers
+#########################
+
+A workflow has one or more controllers, each implementing a specific coordination strategy. For example, the ScatterAndGather
+(SAG) controller implements a popular strategy that is typically used for the fed-average type of federated training. The
+CrossSiteValidation controller implements a strategy to let every client site evaluate every other site's model. You can put together
+a workflow that uses any number of controllers.
+
+Before version 2.4, all federating learning workflows (fed-average, cyclic controller, cross-site evaluation) were server controlled,
```

Comment:
Good we have some explanations for users that have tried old versions.

Should we put these text here or should we move them to release notes? @chesterxgchen ?